src/models/users.ts

```diff
@@ -0,0 +1,40 @@
+import mongoose, { Schema } from 'mongoose';
+
+const usersSchema: Schema = new mongoose.Schema(
+  {
+    name: {
+      type: String
+    },
+
+    userimage: {
+      type: String
+    },
+
+    mail: {
+      type: String,
+      unique: true
+    },
+
+    phone: {
+      type: Number,
```

Comment:
the phone number should be a string if we use a number

this wil happen, because javascript ignore 0 on the left
0960952970 -> 960952970


this also impact on the validators